pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.40.0-dev43"
+__version__ = "0.40.0-dev44"
```

Comment:
Don't forget we must update that (and the changelog) to 0.41 before merging. :slightly_smiling_face: